src/resolvers/Query/getSellerOrders.js

```diff
@@ -0,0 +1,20 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+export default async function getSellerOrders(_, args, context) {
```

Comment:
add following summary for the funcation before declaration according to function 
/**

@name ordersByAccountId
@method
@memberof Order/NoMeteorQueries
@summary Query the Orders collection for orders made by the provided accountId and (optionally) shopIds
@param {Object} context - an object containing the per-request state
@param {Object} params - request parameters
@param {String} params.accountId - Account ID to search orders for
@param {String} params.orderStatus - Workflow status to limit search results
@param {String} params.shopIds - Shop IDs for the shops that owns the orders
@returns {Promise|undefined} - An Array of Order documents, if found
*/